apps/sms/views/shared/test/unit/navigation_test.js

```diff
@@ -127,8 +127,10 @@ suite('navigation >', function() {
     suite('init() >', function() {
       test('display the right panel', function(done) {
         assert.isFalse(elements.InboxView.classList.contains('panel-active'));
+        assert.equal(elements.InboxView.getAttribute('aria-hidden'), 'true');
         Navigation.init().then(() => {
           assert.ok(elements.InboxView.classList.contains('panel-active'));
```

Comment:
ahah I thought you looked at the changed line, sorry.

Contributor filed https://bugzilla.mozilla.org/show_bug.cgi?id=1192636, we  can ask to replace elsewhere if you want. Honestly I don't see much value for this; I like the brevity of `assert.ok` for things where we don't care about the actual boolean value and more about the behavior in a condition.